src/tests/test_main.py

```diff
@@ -0,0 +1,81 @@
+import sys
+import os
+
+from fastapi.testclient import TestClient
+from PIL import Image
+
+from . import TEST_ASSETS_DIR
```

Comment:
There's one failing test. I suggest here:
```suggestion
from . import TEST_ASSETS_DIR, TESTS_DIR
```